### PR TITLE
ConfigParser.parse( ): can return invalid content.

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

File: libert_util/src/util.c [hunk 1]
```diff
@@ -3134,6 +3134,35 @@ bool util_is_first_day_in_month_utc( time_t t) {
...
+
+  The validator will check that the two separation characters are the
+  same.
+*/
+
+bool util_sscanf_isodate(const char * date_token , time_t * t) {
+  int day   , month , year;
+  char sep1 , sep2;
+
+  if (sscanf(date_token , "%d%c%d%c%d" , &year , &sep1 , &month , &sep2 , &day) == 5) {
```
Comment (jepebe) [inline libert_util/src/util.c]: This should probably be: `%4d%c%2d%c%2d` where the only allowed separators are: `-` and nothing for dates. https://no.wikipedia.org/wiki/ISO_8601

Comment (pgdr) [inline libert_util/src/util.c]: "Be conservative in what you do, be liberal in what you accept from others."

Why not accept dots?  Do you foresee any possible problems?

**Comment by pgdr** [comment]
It seems `ert_util_sscan_test` fails:

`ert_util_sscan_test.c:72 => Booleans are different b1:[0]  b2:[1]`

**Comment by joakim-hove** [comment]
Split in #1247, #1248, #1249 and WIP: #1250

**Pull request closed without merging**